ci/docker/buildenv/README.md

```diff
@@ -0,0 +1,60 @@
+# Verilator Build Environment
+
+This container is set up to compile and test a Verilator build based
+on the following parameters:
+
+    - Source repository (default: <https://github.com/verilator/verilator>)
+    - Revision (default: master)
+    - GCC version (4.8.5, 5.5.0, 6.5.0, 7.4.0, default: 7.4.0)
+
+The container is published as `verilator/verilator-buildenv` on
+[docker
+hub](https://hub.docker.com/repository/docker/verilator/verilator-buildenv).
+
+To run the basic build of current master:
+
+```shell
+docker run -ti verilator/verilator-buildenv
+```
+
+To also run tests:
+
+```shell
+docker run -ti verilator/verilator-buildenv test
+```
+
+Change the compiler:
+
+```shell
+docker run -ti -e CC=gcc-4.8 -e CXX=g++-4.8 verilator/verilator-buildenv test
+```
+
+Actually the tests that involve gdb are not working due to security
+restrictions. To run those too:
+
+```shell
+docker run -ti -v ${PWD}:/tmp/repo -e REPO=/tmp/repo -e REV=`git rev-parse --short HEAD` -e CC=gcc-4.8 -e CXX=g++-4.8 --cap-add=SYS_PTRACE --security-opt seccomp=unconfined verilator/verilator-buildenv test
+```
+
+Finally, you may want to avoid pushing your changes to a remote
+repository and instead use a local working copy. You can mount the
+local working copy path as a volume and use this as repo. Be careful,
+that it can only use committed changes, so you may want to use a
+work-in-progress commit or so. To build the current HEAD from top of a
+repository:
+
+```shell
+docker run -ti -v ${PWD}:/tmp/repo -e REPO=/tmp/repo -e REV=`git rev-parse --short HEAD` --cap-add=SYS_PTRACE --security-opt seccomp=unconfined verilator/verilator-buildenv test
+```
+
+## Under the Hood
+
+To rebuild the image, simply run:
+
+```shell
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[code-block-style] Code blocks should be indented](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)

Reply:
Oh, cmon, you are kidding me..

Reply:
;)